airframe-sql/src/main/scala/wvlet/airframe/sql/analyzer/TypeResolver.scala

```diff
@@ -379,7 +385,27 @@ object TypeResolver extends LogSupport {
           case a: Attribute   => a
           case m: MultiColumn => m
           // retain alias for select column
-          case expr => if (isSelectItem) SingleColumn(expr, Some(i.value), None, expr.nodeLocation) else expr
+          case f: FunctionCall =>
```

Comment:
Wondering if the same way can be applied to any expression not only `FunctionCall`. 🤔

Reply:
That's also my concern. Maybe converting every type of expression as ResolvedAttribute?

Reply:
I'm not 100% but maybe that should be.

Reply:
Marked as TODO for now

Reply:
Resolved every matched expressions as ResolvedAttribute here